packages/salesforcedx-vscode-core/package.json

```diff
@@ -128,6 +135,11 @@
           "command": "sfdx.force.apex.trigger.create",
           "when":
             "explorerResourceIsFolder && resourceFilename != aura && resourceFilename != lightningcomponents && sfdx:project_opened"
+        },
+        {
+          "command": "sfdx.force.source.retrieve",
```

Comment:
The command also appears in the dropdown menu when a user right-clicks on a file from the Explorer side-panel